modules/apps/headless-apio/forms/forms-apio-impl/src/main/java/com/liferay/forms/apio/internal/architect/resource/FormInstanceRecordNestedCollectionResource.java

```diff
@@ -210,30 +211,18 @@ private Long _extractFileEntryId(DDMFormFieldValue ddmFormFieldValue) {
 	}
 
 	private PageItems<DDMFormInstanceRecord> _getPageItems(
-		Pagination pagination, Long formInstanceId) {
-
-		Integer count = Try.fromFallible(
-			() -> _ddmFormInstanceRecordService.getFormInstanceRecordsCount(
-				formInstanceId)
-		).orElse(
-			null
-		);
-
-		if (count == null) {
-			return null;
-		}
+		Pagination pagination, Long formInstanceId) throws PortalException {
 
-		return Try.fromFallible(
-			() -> _ddmFormInstanceRecordService.getFormInstanceRecords(
+		List<DDMFormInstanceRecord> ddmFormInstanceRecords =
+			_ddmFormInstanceRecordService.getFormInstanceRecords(
 				formInstanceId, WorkflowConstants.STATUS_ANY,
```

Comment:
STATUS_ANY ¿?

Reply:
if (status == WorkflowConstants.STATUS_ANY) {
				sql = StringUtil.replace(
					sql, "(DDMFormInstanceRecordVersion.status = ?) AND",
					StringPool.BLANK);
			}